scrapers/perfectgonzo.yml

```diff
@@ -0,0 +1,28 @@
+name: Perfect Gonzo
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - perfectgonzo.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="col-sm-6 col-md-6 no-padding-left no-padding-right"]/h2
+      Date:
+        selector: //div[@class="col-sm-6 col-md-6 no-padding-left no-padding-right text-right"]/span
+        postProcess:
+          - replace:
+              - regex: 'Added|,'
+                with: ''
+          - parseDate: January 02 2006
+      Details: //div[@class="col-sm-8 col-md-8 no-padding-side"]/p
+      Tags:
+        Name: //div[@class="col-sm-8 col-md-8 no-padding-side tag-container"]//a
+      Studio:
+        Name:
+          fixed: Perfect Gonzo
+      Image: //video[@id="video"]/@poster | //div[@class="container bloc-sm"]//@src
```

Comment:
Do you have the URL of the scene that the second part is needed?
The second part should be more restrictive as it seems to return a lot of links inlcluding videos

Reply:
Any scene that is an interview as a different image layout.
[https://www.perfectgonzo.com/movies/dolly-diore-interview/?nats=](url)
Having a look it should have been 
class="container col-sm-12"

Reply:
```yaml
      Image: //video[@id="video"]/@poster | //div[@id="video-hero"]//img/@src
```
seems to get the job done and is probably more future proof
can you verify against some of your samples ? (works for me for a few scenes)

Reply:
Yup works on all scenes , you can make the change :-)